api_app/analyzers_manager/views.py

```diff
@@ -46,6 +47,7 @@ class AnalyzerListAPI(APIView):
     def get(self, request, *args, **kwargs):
         try:
             ac = self.serializer_class.read_and_verify_config()
+            CustomConfig.apply(ac, request.user, CustomConfig.PluginType.ANALYZER)
```

Comment:
cool, in this way you can do a unit test for this function.

* choose a possible input
* call the function
* check if the data generated/changes is the same of what you expect (use `self.assert`-like clauses from the `unittest` framework`

Reply:
@devmrfitz I just checked the usages of `CustomConfig.apply` and I did not find anything on tests for that. Can you make a test as Matteo asked? After that, I think we are good to go with this PR

Reply:
Done